tests/test_Guards.py

```diff
@@ -35,7 +36,114 @@ def test_Guards__safe_builtins__2(e_exec):
         __name__='restricted_module',
         __metaclass__=type,
         _write_=lambda x: x,
-        _getattr_=getattr)
+        _getattr_=getattr,)
 
     e_exec(CLASS_SOURCE, restricted_globals)
     assert restricted_globals['b'] == '2411'
+
+
+class D:
+    value = None
+    _guarded_writes = 1
+
+
+SET_ATTRIBUTE = '''
+d1 = D()
+setattr(d1, 'value', 9999)
+'''
+
+
+DEL_ATTRIBUTE = '''
+delattr(d1, 'value')
+'''
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_Guards__safe_builtins__3(e_exec):
```

Comment:
I think it should be called `test_Guards__guarded_setattr__1` as it actually tests `guarded_setattr`.